chainer/functions/evaluation/binary_accuracy.py

```diff
@@ -36,16 +36,44 @@ def binary_accuracy(y, t):
     """Computes binary classification accuracy of the minibatch.
 
     Args:
-        y (Variable): Variable holding a matrix whose i-th element
-            indicates the score of positive at the i-th example.
+        y (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Array whose i-th element indicates the score of
+            positive at the i-th sample.
+            The prediction label :math:`\\hat t[i]` is ``1`` if
+            ``y[i] >= 0``, otherwise ``0``.
+
         t (Variable): Variable holding an int32 vector of ground truth labels.
```

Comment:
It seems that ``t`` can also be ``np.ndarray`` or ``cupy.ndarray`` like ``y``.
Please add these types for  the documentation.

Reply:
Fixed